v2/manifests/core-ons/zebedee.yml

```
@@ -5,6 +5,10 @@ services:
     build:
       context: ${DP_REPO_DIR:-../../../..}/zebedee
       dockerfile: Dockerfile
+      args:
+        # if you want to change this port you should also change the mapping for it below in the ports section
+        # and rebuild the docker image
```

I think it would be useful to say which of the ports in the ports section, host port and/or container port, will need to be changed? E.g. (if it's both)
```suggestion
        # so that both the host port and container port use the new port number, and then rebuild the docker image
```

---

This is now addressed.